UltiSnips/elixir.snippets

```diff
@@ -0,0 +1,126 @@
+snippet if
+if ${1:condition} do
+	${2:expression}
+${3:end}
```

Comment:
If you wanted an if/else, you can just replace `end` with `else` and hit `<tab>` to get it.